tests/acceptance/pageObjects/publicLinkPasswordPage.js

```diff
@@ -36,21 +36,25 @@ module.exports = {
           .waitForElementPresent({ selector: '@filesListProgressBar', abortOnFailure: false }) // don't fail if we are too late
           .waitForElementNotPresent('@filesListProgressBar')
       },
-      assertResourceAccessDenied: function () {
-        return this
+      /**
+       * gets resource access denied message after clicking submit password button for a public link share
```

Comment:
Would be nice to have clicking and getting status message in different functions.

Reply:
done!
:rocket: